.github/workflows/ci.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    branches:
+      - master
+      - release
+  pull_request:
+name: ci
+jobs:
+  run:
```

Comment:
Maybe call this job `test` since it runs the CI tests (as opposed to the CI deploy, for example)?

Reply:
Done.